package/kernel/qca-ssdk/Makefile

```
@@ -58,6 +58,7 @@ endif
 
 ifeq ($(CONFIG_TARGET_SUBTARGET), "ipq60xx")
     MAKE_FLAGS+= CHIP_TYPE=CPPE
+    MAKE_FLAGS+= IN_AQUANTIA_PHY=TRUE
```

```suggestion
    IN_AQUANTIA_PHY=TRUE
```

Then there is no need for `MAKE_FLAGS` again